pkgs/tools/filesystems/zxfer/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub
+, coreutils, gnused, gawk, gnugrep
+, openssh, rsync
+, gzip }:
```

One item per line is nowadays the preferred way to do as it creates simpler diffs.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, coreutils
, gnused
, gawk
, gnugrep
, openssh
, rsync
, gzip
}:
```